src/events/binaryEvent.ts

```diff
@@ -1,22 +1,22 @@
 import { v4 as uuid } from "uuid";
 
-export interface BinaryEventData {
+export interface BinaryEventData<T extends string = string> {
   id: string;
   contentType: "application/octet-stream";
   eventType: string;
```

Comment:
Should be `T`

Reply:
Fixed in 4bcb16755e195d48d4959cb7b69987b7cbdd3cda